discord_bots/utils.py

```diff
@@ -137,13 +137,9 @@ def create_finished_game_embed(
     user_name: Optional[str] = None,
 ) -> Embed:
     # assumes that the FinishedGamePlayers have already been comitted
-    aware_db_datetime: datetime = finished_game.finished_at.replace(
-        tzinfo=timezone.utc
-    )  # timezones aren't stored in the DB, so add it ourselves
     embed = Embed(
         title=f":white_check_mark: Game '{finished_game.queue_name}' ({short_uuid(finished_game.game_id)})",
```

Comment:
here you can copy/paste the checkmark emoji directly. I forgot to change it here

Reply:
roger, went ahead and changed that too.